packets.go

```diff
@@ -525,6 +532,13 @@ func (mc *mysqlConn) handleErrorPacket(data []byte) error {
 		pos = 9
 	}
 
+	//If error code is for Connection was killed, then return bad connection.
```

Comment:
Also add a space here and put "Connection was killed" in quotes. Maybe also replace "then return bad connection" with "then mark connection as bad"